frontend/elements/src/entry/jig/publish/panel/jig-panel.ts

```diff
@@ -0,0 +1,100 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import "@elements/core/buttons/text";
+
+
+@customElement('jig-panel')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+    
+    
+   main{
+    width: 416px;
+    height: 1402px;
+    position:ralative;
+
+   }
+  
+.jig-head{ 
+   width: 150px;
+   height: 182.1px;
```

Comment:
image here should not have set width and height. it should have a width and height should be auto but I believe the image here is the correct size and doesnt need height or width